pom.xml

```diff
@@ -105,8 +106,6 @@
     <!-- Release: enable publication to Bintray -->
     <artifactsToPublish>${project.groupId}:sonar-php-plugin:jar</artifactsToPublish>
 
-    <!-- FIXME fix javadoc errors and remove this -->
-    <doclint>none</doclint>
```

Comment:
Is this fixed? I remember there was an impact on one of our products.

Reply:
ok, reverted